app/builders/subscription_auth_email_builder.rb

```diff
@@ -13,47 +14,42 @@ def call
     Email.create!(
       subject: subject,
       body: body,
-      address: subscriber.address,
-      subscriber_id: subscriber.id,
-    )
+      address: address,
+      )
   end
 
   private_class_method :new
 
 private
 
-  attr_reader :subscriber, :destination, :token
+  attr_reader :address, :token, :frequency, :topic_id
 
   def subject
-    "Confirm your email address"
+    "Confirm your subscription"
   end
 
   def body
     <<~BODY
-      # Click the link to confirm your email address
+      # Click the link to confirm your subscription
 
-      ^ [Confirm your email address](#{link})
+      ^[Confirm your subscription](#{link})^
 
-      You need to do this to manage your GOV.UK email subscriptions. The link will stop working in 7 days.
+      This link will stop working in 7 days.
 
-      # Didn’t request this email?
+      **Didn’t request this email?**
 
-      Ignore or delete this email if you didn’t request it. Your subscriptions won’t be changed.
+      Ignore or delete this email if you didn’t request it.
+
+      [Read our privacy policy (opens in a new tab)](https://www.gov.uk/help/privacy-notice) to find out how we use and protect your data.
```

Comment:
It might just be me, but the `opens in a new tab` copy is a bit weird. Are we sure that's right?

Reply:
Yes it is weird and I already flagged it. It is considered part of MVP and is going to be iterater